<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRackContainer</class>
 <widget class="QScrollArea" name="CRackContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrollArea</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CParametersContainer" name="ParametersContainer" native="true"/>
    </item>
    <item>
     <widget class="CJacksComponent" name="JacksComponent">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>112</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>JacksComponent</zorder>
   <zorder>ParametersContainer</zorder>
   <zorder>JacksComponent</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CParametersContainer</class>
   <extends>QWidget</extends>
   <header>cparameterscontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CJacksComponent</class>
   <extends>QGraphicsView</extends>
   <header>cjackscomponent.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
